### DISCUSSION TITLE
Replacement/workaround for brew switch?

### DISCUSSION THREAD

**Discussion by briantully** [Homebrew/discussions#1052] [Everyday usage]
I'm trying to switch back to an older version of a keg that I already have installed where I realized the new version of the keg (jmeter) is buggy.

I used to be able to do "brew switch" but this appears to have been removed?

Is there a workaround or replacement for "brew switch"? I saw a different discussion thread that seemed to highlight this without a clear workaround, and for some reason that discussion thread is locked/closed.

Any help would be greatly appreciated.

**Comment by SMillerDev** [comment]
There is no replacement. If you want to install a specific version you'll have to `brew extract`

**Reply by briantully** [reply]
Thank you @SMillerDev! `brew extract` is exactly what I needed :)

**Comment by laggardkernel** [comment]
Included in my tap.

```bash
brew tap laggardkernel/tap
brew switch --help
```

I don't use `brew extract`, it's different from `brew switch` and serves different purpose.
